### ISSUE TITLE
What program do you use to compile?

### ISSUE THREAD

**Issue by Vitae666** [MatthieuBonne/CQMacroCreator#7]
Just wondering, if I wanted to take a look at doing some modifications and then testing those, what do I need on my laptop and how do I then compile it?
I'm still thinking about the whole thing that I babbled on the CQA thread about the needing to change the display settings to have it not look weird.
Wanted to see if I could figure it out on my own.

**Comment by MatthieuBonne** [comment]
I use Visual Studio : https://visualstudio.microsoft.com/fr/downloads/

**Comment by Vitae666** [comment]
Whelp, I wasted a whole day downloading and installing this. Shitty internet. Downloads are about 120Kb/s if I'm lucky. Fucking AT&T.
Have no idea what to do. 
When I opened CQMacroCreator.sln the 1st time it gave me something about needing to download .NET desktop development. That was about 3GB large, so I do it.
After it rebooted my laptop, I tried again and now it gives some error about looking for one version of Net Framework when it's already there but apparently typoed somewhere so I download 4.6.1 and install it hoping that will work. Nope!
Open CQMacroCreator.sln again and get the same error. I tell it to change it this time.
![image](https://user-images.githubusercontent.com/9310914/63483463-76c96b80-c46a-11e9-8d2c-4cc48a754ba9.png)
_Note it says its looking for NetFramework and asks to change it to Net Framework._

I made the change to the file that I wanted to do the test with, tried to build it and got a load of stuff saying that something or other didn't exist. Including the Newston.json which I know is there...
![image](https://user-images.githubusercontent.com/9310914/63483588-ea6b7880-c46a-11e9-9a1d-c4c6cfbfe1d1.png)

Really, all I wanted to do was check to see if changing line 22807 in Form1.Designer.cs from
this.AutoScaleMode = System.Windows.Forms.AutoScaleMode.Font;
to
this.AutoScaleMode = System.Windows.Forms.AutoScaleMode.Dpi;
or
maybe Inherit or None would help that problem I have with viewing CQMC correctly.
Saw a mention of it on
https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.autoscalemode?view=netframework-4.6.1

But I obviously have no clue what I'm doing to even be able to create a build (exe?) correctly. 

So, started downloading from scratch today @ around 10AM and it's now 11PM
MEH! :-(

**Comment by Vitae666** [comment]
OK! I might have gotten a step further in all this. I saw the Tools dropdown with "Get Tools and Features" so, I clicked it and it gave me a screen where I could download Individual Components like .NET Framework SDK and targeting pack. So I downloaded those.
Reloaded CQMacroCreator.sln and NO opening screen error! I cheered!
Then I tried a Build, and it gave me:

Error		Couldn't process file Form1.resx due to its being in the Internet or Restricted zone or having the mark of the web on the file. Remove the mark of the web if you want to process these files.

Da'hell? "Mark of the web"??

So, I found something that said to remove <data name="$this.Icon" ... /data> in Form1.resx
Did that. Did a Build SUCCESSFULLY!....or not...Dpi made the screen huge. I have 2 monitors and it went across both and only showed 2 heroes across and 6 down. It's a mess. 

Changed to Inherit and it looks exactly like it did with Font. Tried None and still looks like Font.
Dammit! Was hoping that was the actual solution! :(

Of course, if I WERE to try to make changes somehow, I don't even know if a Build would work for anyone because of that Mark of the Web thing. 
But, have a feeling I won't be doing anything at all. 
If I fix it on my screen, 99% sure that anyone who already can see it the way it's supposed to be with how Alya has it, will have it totally screwed on their screen cause of my changes.

**Comment by MatthieuBonne** [comment]
Wow, that was complicated. ^^ Some info about the "mark of the web" : https://developercommunity.visualstudio.com/content/problem/291761/couldnt-process-file-abcresx-due-to-its-being-in-t.html
Dunno what to do about your display issue though...

**Comment by Vitae666** [comment]
Complicated is my life! LOL

That's actually exactly where I found the answer I used.
It said to remove ALL <data ... /data> nodes, but the example he gave was for the $this.Icon so I did that 1st and tried and it worked. Didn't want to screw with the other data ones cause they looked important :)

I might start to screw around with the sizing or something and see what happens. BUT I'd need someone to test it that sees it correctly already. Just to see if it's OK on their side or screwed worse.
